docs/installation/docker.mdx

```diff
@@ -47,6 +47,12 @@ Diese Dokumentation ist für Docker und Synology. Bei Verwendung anderer Systeme
 
 Der Docker Container kann entweder manuell über einen Befehl in der Kommandozeile erstellt werden, oder über eine `docker-compose` Datei. Wem letzteres nichts sagt, führt die Schritte unter _Manuell_ aus.
 
+Die Einrichtung von evcc über Docker auf der Synology NAS ist über zwei Varianten möglich, die abhängig davon sind, ob die Wallbox den EEBus verwendet. Dies ist z.B. bei der Elli Charger Pro (oder Connect mit nachgerüstetem Energiemesser) der Fall. Der Docker Container muss dann im Host-Mode eingerichtet werden. Dies ist hier beschrieben:
+[Anleitung.EVCC.Synology.Docker.Elli.Charger.Connect-Pro.pdf](https://github.com/evcc-io/docs/files/10365841/Anleitung.EVCC.Synology.Docker.Elli.Charger.Connect-Pro.pdf)
+
+In anderen Fällen wird der Bridge-Mode empfohlen. Hier ist eine Anleitung dazu: [EVCC_Synology_Docker-2.pdf](https://github.com/evcc-io/docs/files/10365845/EVCC_Synology_Docker-2.pdf) (erstellt von at4hawo1)
```

Comment:
Das stimmt so nicht und leitet in die Irre. Es gibt verschiedenste Gründe für HOST mode, z.B. die UDP Kommunikation der KEBA, mDNS allgemein wenn man das nutzen möchte etc.